store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -5851,6 +5863,48 @@ public AddInviteData addInvite(OperationContext octxt, Invite inv, int folderId,
         }
     }
 
+    public String getMsgHtmlDesc(ParsedMessage pm) throws ServiceException {
+        String htmlDesc = null;
+        if (pm == null)
+            return htmlDesc;
+
+        Account acct = getAccount();
+        MimeMessage mm = pm.getMimeMessage();
+        List<MPartInfo> parts;
```

Comment:
nit: null assignment.

Reply:
null assignment shouldn't be needed as in this case its empty and its being checked for empty list in the code below.

Reply:
Technically it looks good. That's why mentioned as nit. As discussed let's do it.